learn-express-build-strock-trading-platform/server050.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, function() {
+  console.log('Your app is listening on port 3000.');
+});
+
+app.get('/hi', (req, res) => {
+  res.send('Hi there!');
+});
+
+const prices = {
+  ABC: 48.83,
+  DEF: 2.98,
+  GHI: 3.99,
+  JKL: 99,
+  MNO: 45.38,
+  OPQ: 0.48,
+  RST: 9.32,
+  UVW: 10.94,
+  XYZ: 5.32,
+};
+
+// 050.  Let create our /buy endpoint.
```

Comment:
This could be broken down further. One step to write `app.get('/buy')`, the next to pass it the callback function, and then two more lessons to explain route parameters `:ticker` and `:shares`.